tbx/people/models.py

```diff
@@ -7,13 +7,11 @@
 from modelcluster.fields import ParentalKey
 from modelcluster.models import ClusterableModel
 from phonenumber_field.modelfields import PhoneNumberField
-from tbx.blog.models import BlogIndexPage, BlogPage
+from tbx.blog.models import BlogPage
 from tbx.core.blocks import PageSectionStoryBlock
 from tbx.core.utils.models import SocialFields
-from tbx.people.blocks import InstagramEmbedBlock, StandoutItemsBlock
```

Comment:
the StandoutItemsBlock and InstagramEmbedBlock could now be removed along with any related templates if they are not used elsewhere

Reply:
Thanks, removed in f925b0a and 95de04c